lib/uniqueness/model.rb

```diff
@@ -60,7 +60,8 @@ def uniqueness_validation
           end
           conditions[field] = value
           query = self.class.where(conditions)
-          errors.add(field, 'should be unique') if query.any?
+          results = query.all - [self]
```

Comment:
I'd go for adding to `conditions` instead of this.

Reply:
Hi @owahab, I know that this is an old PR and comment, but would you be able to shed any more light on the use of conditions?

I've attempted to add a condition via the `scope: []` option in the `options` param, but it both appends the scope query to `conditions` and the name of the scope to the `query`, leading to a DB adapter error (Postgres in my situation). The use of `send()` causes a `NoMethodError` if relying purely on a scope.

I've searched around for this pattern to try an obtain better knowledge of how it works (maybe I'm implementing incorrectly), but it has been in vein.  Any guidance would be greatly received, an example added to the README would be even better 😃.